test/evergreen.yml

```diff
@@ -5830,7 +5836,7 @@ buildvariants:
   - ubuntu2004-arm64-large
   batchtime: 1440 # 24 hours
   expansions:
-    ENABLE_TCMALLOC: 1
+    ENABLE_TCMALLOC: 0
```

Comment:
Please kindly remind me why we need to disable tcmalloc for this builder (it's not a sanitizer builder).

Reply:
Could well be wrong. I thought it was used as a pre-req for sanitizer builds defined later `evergreen.yml`.